### ISSUE TITLE
conda activate completion

### ISSUE THREAD

**Issue by harelwa** [esc/conda-zsh-completion#35]
## Issue
`conda activate` does not complete existing _named_ envs other then `base`.
```bash
(base) > $  # Pressing `tab` after `conda activate` with _no_ prefix yields --
(base) > $ conda activate base base base base base base base base base # etc. 

(base) > $ # Pressing `tab` after `conda activate` with _something_ prefix yields --
(base) > $ conda activate something_main_complete:202: permission denied:
_main_complete:202: permission denied:
_main_complete:202: permission denied:


(base) > $ conda env list  # other envs exist
# conda environments:
#
ishow-oxip                        /homes/harelw/.conda/envs/ishow-oxip
py3.7-mx-build                 /homes/harelw/.conda/envs/py3.7-mx-build
py3.7-opencv                    /homes/harelw/.conda/envs/py3.7-opencv
py3.7-oxip-mx-test           /homes/harelw/.conda/envs/py3.7-oxip-mx-test
py3.7-oxucicd-mx-dev      /homes/harelw/.conda/envs/py3.7-oxucicd-mx-dev
base                              *   /usr/local/conda

(base) > $ cat  ~/.conda/environments.txt  # also here, for completeness 
/homes/harelw/.conda/envs/py3.7-oxip-mx-test
/homes/harelw/.conda/envs/ishow-oxip
/homes/harelw/.conda/envs/py3.7-mx-build
/homes/harelw/.conda/envs/py3.7-oxucicd-mx-dev
/homes/harelw/.conda/envs/py3.7-opencv


(base) > $ # other compeltions work
(base) > $ conda env # pressed tab
create  -- Create an environment based on an environment file
export  -- Export a given environment
list    -- List the Conda environments
remove  -- Remove an environment
update  -- Update the current environment based on environment file
```

## Environment and Installation Details
- Using `oh-my-zsh` ,
- Plugin installed according to instructions in `_conda`:
  ```bash
  git clone https://github.com/esc/conda-zsh-completion ${ZSH_CUSTOM:=~/.oh-my-zsh/custom}/plugins/conda-zsh-completion
  ```
- Plugin added to `~/.zshrc`
- Tried with both `autoload` after / before `source $ZSH/oh-my-zsh.sh`
- _conda version_ `4.8.2` 
  - `miniconda` installation
- _os_ `ubuntu 18.04`

Many thanks,
Harel

**Comment by huyz-git** [comment]
Change
https://github.com/esc/conda-zsh-completion/blob/766e4a2d8f1844dad28d10f5c3364fafe0e13348/_conda#L141
into
```
local -a envs tmp
```
Then the `permission denied` issue will fixed
For the first issue, you can add `zstyle ":conda_zsh_completion:*" show-unnamed true` into your `.zshrc` file to show it. Or try https://github.com/esc/conda-zsh-completion/pull/36

**Comment by harelwa** [comment]
many thank @huyz-git ! 
I applied both changes both problems are solved.

**Comment by sawyermade** [comment]
Why hasn't this been pushed to the master yet?

**Comment by esc** [comment]
@sawyermade thanks for the ping, the original notification seems to have never made it into my inbox.

**Comment by esc** [comment]
fixed and improved upon by #36
